.azure-pipelines-canary.yml

```diff
@@ -24,10 +24,11 @@ stages:
       androidVariableGroup: 'ApplicationTemplate.Distribution.Internal.Android'
       iosProvisioningProfileFile: $(InternalProvisioningProfile)
       iosCertificateFile: $(InternalCertificate)
-      iosVariableGroup: 'ApplicationTemplate.Distribution.Internal.iOS'
+      iosVariableGroup: 'ApplicationTemplate.Distribution.Internal.iOS.TestFlight'
```

Comment:
Why add ".TestFlight"? It implies that there's another group that's iOS and not TestFlight.
It also doesn't match the documentation in variables.yml.